package.json

```diff
@@ -26,7 +26,8 @@
     "typecheck": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false",
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "release": "vite build && storybook build"
```

Comment:
Since our build is currently just using Storybook's process, we might just want to use the `dist` files created in `storybook-static`. If we need to move them, we could potentially add that to the workflow script.